default/templates/hooks/more-head.tpl

```diff
@@ -34,4 +34,9 @@
   .header-font {
     font-family: 'MavenPro', sans-serif;
   }
+
+  .embedded-pdf {
+    width: 100%;
+    height: 800px;
+  }
```

Comment:
You should remove these in favour of inline styles in `embed-pdf.tpl`. Prefer Tailwind classes wherever possible; for example, use `w-full` instead of `width: 100%;`.